src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -87,6 +87,7 @@
         "wake-on-lan-cluster.xml",
         "wifi-network-diagnostics-cluster.xml",
         "window-covering.xml",
+        "replacable-monitoring-cluster.xml",
```

Comment:
Probably better to add it closer to the right spot alphabetically in the cluster list above this point.